js/coinbasepro.js

```diff
@@ -879,6 +879,9 @@ module.exports = class coinbasepro extends Exchange {
         if (type === 'withdraw') {
             type = 'withdrawal';
             address = this.safeString (details, 'sent_to_address', address);
+            if (typeof details.fee === 'string') {
```

Comment:
This won't work, unfortunately, cause it breaks the following rules:

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#multilanguage-support
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#derived-exchange-classes

Also, the fee must be a structure, with a code and cost, not a scalar number.
But anyways, thank you for your involvement, I will upload a proper fix in a shortly.

Reply:
Oh, sorry. I missed that. Thank's for your help!